hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/file/OMDirectoryCreateRequest.java

```diff
@@ -210,7 +210,7 @@ public OMClientResponse validateAndUpdateCache(OzoneManager ozoneManager,
 
   private OmKeyInfo createDirectoryKeyInfo(OzoneManager ozoneManager,
       OmBucketInfo omBucketInfo, String volumeName, String bucketName,
-      String keyName, KeyArgs keyArgs)
+      String keyName, KeyArgs keyArgs, long transactionLogIndex)
```

Comment:
Is the transaction log index reset when the Ratis term changes?

Reply:
No Arpit. Transaction log index is the ratis log index. It is monotonically increasing always.